ecleankernel/layout/blspec.py

```diff
@@ -70,37 +78,70 @@ def find_kernels(self,
             exclusions=exclusions,
             module_directory=self.root / 'lib/modules')
 
-        # collect from /boot
```

Comment:
There's clearly a lot of duplication here, and reindenting + all the changes in the single commit make the diff really hard to read.

Reply:
I can look at this again to see if maybe some of the duplication can go. Not really sure what I can do about the diff being hard to read.

Reply:
Now with less code duplication, at the cost of adding one extra function.

I think the readability of the patch has somewhat improved but it is still a bit difficult because of the indentation changes.

Let me know what you think.